lib/howl/modes/default_mode.moon

```diff
@@ -14,13 +14,13 @@ class DefaultMode
 
   code_blocks: {}
 
-  indent: (editor) =>
+  indent: (editor, lines=editor.active_lines) =>
     indent_level = editor.buffer.config.indent
     dont_indent_styles = comment: true, string: true
     buffer = editor.buffer
     current_line = editor.current_line
 
-    editor\transform_active_lines (lines) ->
+    editor\transform_lines lines, ->
```

Comment:
@nilnor 

>  but given the heuristic of using the mode for the first line for mixed selections (which I approve of), then this only seems complicated.

Good point. This was for the original version. I'll try to revert this.